docs/site/content/repo-docs/reference/run.mdx

```diff
@@ -369,11 +369,11 @@ By default, `turbo` will use `grouped` logs in CI environments and `stream` logs
 turbo run build --log-order=stream
 ```
 
-| Option    | Description                               |
-| --------- | ----------------------------------------- |
-| `stream`  | Show output as soon as it is available    |
-| `grouped` | Group output by task                      |
-| `auto`    | Turbo decides based on its own heuristics |
+| Option    | Description                                   |
+| --------- | --------------------------------------------- |
+| `stream`  | Show output as soon as it is available        |
+| `grouped` | Group output by task                          |
+| `auto`    | Turborepo decides based on its own heuristics |
```

Comment:
We should use `turbo` here since we're referring to the CLI (and we're doing the same in the paragraph above this table)